src/pages/index.tsx

```diff
@@ -39,12 +39,26 @@ const AdminPage: NextComponentType = () => {
     },
   });
 
+  async function getAndSetCount(): Promise<void> {
+    const res = await fetch('/api/supportRequest/getCount');
+    if (!res.ok) {
+      setError(true);
+      return;
+    }
+
+    try {
+      setCounts(await res.json());
+    } catch (err) {
+      setError(true);
+    }
+  }
+
   React.useEffect(() => {
     formik.setFieldValue('adminName', window.localStorage.getItem(ADMIN_NAME_KEY));
 
     const timer = setInterval(() => {
       setLastUpdateEpoch(Date.now());
-    }, 30000);
+    }, 3000);
```

Comment:
3 seconds is too fast, let's settle for a good middle ground of 10s
```suggestion
    }, 10000);
```